contrib/Makefile.autosetup

```diff
@@ -1,7 +1,7 @@
-SAMPLES=	colors.default colors.linux gpg.rc Mush.rc Pine.rc \
-		sample.mailcap sample.neomuttrc sample.neomuttrc-starter sample.neomuttrc-tlr smime.rc \
-		smime_keys_test.pl Tin.rc mairix_filter.pl
-
+SAMPLES=	colors.default colors.linux gpg.rc mairix_filter.p Mush.rc Pine.rc \
+		sample.mailcap sample.neomuttrc sample.neomuttrc-starter \
+		sample.neomuttrc-tlr smime.rc smime_keys_test.pl Tin.rc 
+ 
```

Comment:
A couple of trailing space chars, here.

Reply:
Just removed them:

https://github.com/neomutt/neomutt/compare/849c16d6f8917a37bb237a9ab9e22f3f45c17d90..deb302a74aeb0ddb03ca13a3ce4ef6c54d4eec28